metaflow/plugins/list_cli.py

```diff
@@ -0,0 +1,69 @@
+from metaflow._vendor import click
+from metaflow import Flow
+from metaflow.exception import MetaflowNotFound
+import json
+
+
+@click.group()
+def cli():
+    pass
+
+
+@cli.group(help="List objects pertaining to your flow.")
+def list():
+    pass
+
+
+def _fetch_runs(flow_name, num_runs):
+    counter = 1
+    try:
+        flow = Flow(flow_name)
+    except MetaflowNotFound:
+        flow = None
+    run_list = []
+    if flow:
+        for run in Flow(flow_name).runs():
+            if counter > num_runs:
+                break
+            counter += 1
+            run_list.append(
+                dict(
+                    created=str(run.created_at),
+                    name=flow_name,
+                    id=run.id,
+                    status=run.successful,
+                    finished=run.finished,
+                )
+            )
+    return run_list
+
+
+@click.option(
+    "--num-runs",
+    default=10,
+    help="Number of runs to show.",
+)
+@click.option(
+    "--as-json",
+    default=False,
+    is_flag=True,
+    help="Print run list as a JSON object",
+)
+@click.option(
+    "--file",
+    default=None,
+    help="Save the run list to file as json.",
+)
+@list.command(help="List recent runs for your flow.")
+@click.pass_context
+def runs(ctx, num_runs, as_json, file):
+    run_list = _fetch_runs(ctx.obj.flow.name, num_runs)
+    if not run_list:
+        ctx.obj.echo("No runs found for flow: {name}".format(name=ctx.obj.flow.name))
```

Comment:
Nit: this message is incorrect (potentially) if num_runs <= 0

Reply:
I thought [empty lists are falsy](https://docs.python.org/3/library/stdtypes.html#truth-value-testing)?  I have a feeling I'm missing something else, just trying to learn, thanks.

Reply:
Ohh you mean if the cli argument itself is <= 0 !!!! I didn't think about this.  Okay

Reply:
ok I fixed this via https://github.com/Netflix/metaflow/pull/1037/commits/fbba6be401b9f4a50f3accaabcf20b85dad4d443